### PR TITLE
Order Book UI updates

### PR DESCRIPTION
**Pull request by EvgeniiaVak**
refs #103

### CODE REVIEW

File: src/app/components/OrderBook.tsx [hunk 1]
```diff
@@ -109,31 +113,36 @@ export function OrderBook() {
...
-        <div className="text-end">
-          Price
-          <br />({token2Symbol})
-        </div>
-        <div className="text-end">
-          Size
-          <br />({token1Symbol})
+    <div className="p-2 text-sx text-primary-content">
+      <div className="sized-columns">
+        <div className="mx-2 my-4 whitespace-nowrap text-secondary-content">
```
Comment (abdulhaseeb13mar) [inline src/app/components/OrderBook.tsx:118]: @EvgeniiaVak we do not need Order Book now, it was just placed by me for understanding the placement of the layout component

Comment (EvgeniiaVak) [reply src/app/components/OrderBook.tsx:118]: I took it from Vlad's figma. Also, I think I like it, but we can remove of course. Did you try with and without it, what looks better to you?

Comment (EvgeniiaVak) [reply src/app/components/OrderBook.tsx:118]: ok, removing, it's ok without it too. Oh, also I forgot the narrow screen fix, back to draft for now.

Comment (abdulhaseeb13mar) [reply src/app/components/OrderBook.tsx:118]: i think the order book component is pretty much self explanatory to the users. and besides we are not giving headings to other components. so i would suggest to keep it consistent with other components

Comment (abdulhaseeb13mar) [reply src/app/components/OrderBook.tsx:118]: > i think the order book component is pretty much self explanatory to the users. and besides we are not giving headings to other components. so i would suggest to keep it consistent with other components

referring to this figma design 
image

**Comment by EvgeniiaVak** [comment]
Atm I don't have much time for making the nice collapsible thing for mobile screens, but at least orderbook is visible now there too.

**Review by abdulhaseeb13mar — APPROVED** [review]

**Comment by abdulhaseeb13mar** [comment]
@EvgeniiaVak Looks good to me you can merge now

**Pull request merged**